docs/backplane/requirements/acs/10-acs-managed-service.md

```diff
@@ -0,0 +1,41 @@
+# *ACS Team* for *OSD/ROSA*
+
+Process is captured [ROX-22017: Harden Network Security API Permissions](https://docs.google.com/document/d/1lyzFjK51py6o62zS5ErPFLbCNVJfq4e5PvVI2Y2z0Mg/edit)
+
+label selectors:
+* `api.openshift.com/addon-acs-fleetshard: "true"` OR
+* `api.openshift.com/addon-acs-fleetshard-qa: "true"` OR
+* `api.openshift.com/addon-acs-fleetshard-dev: "true"` 
+
+for where access is applied
+
+These permissions are a baseline acess for the ACS:CS team across OSD/ROSA infra.
+
+## backlplane-acs-admins-cluster: `cluster`
+Permissions at cluster scope.
+
+* view machine resources
+* view ingresscontrollers
+* view ingresses
+* view nodes
+* view autoscaling
+
+## backplane-acs-admins-project: `(^redhat-acs-fleetshard$|^rhacs$|^rhacs-.*)`
+ACS team needs read/list/watch access to core `redhat` and `rhacs` objects within their namespaces.
+
+Permisions at namespace scope.
+
+* get centrals
+* get stackrox
+* get secureclusters
+* get pod logs
+* get configmaps
+* view network policies
+* scale machinesets
+* view monitoring
+* get routes
+* get deployments
+* get pods
+* scale prometheus
+* resize and delete PVCs
```

Comment:
Don't think they should have PVC deletion card blanche

Reply:
No, I would that they should have this access actually, I am currently in the process of reviewing their additional RBAC requests linked in this document.